src/twisted/runner/test/test_procmon.py

```diff
@@ -219,6 +227,19 @@ def test_startProcess(self):
         self.assertIsInstance(self.pm.protocols["foo"], LoggingProtocol)
         self.assertIn("foo", self.pm.timeStarted.keys())
 
+    def test_startProcessAlternateLoggingProtocol(self):
+        """
+        Allow starting a process with a custom
+        L{twisted.internet.interfaces.ILoggingProcessProtocol}.
+        """
+        @implementer(interfaces.ILoggingProcessProtocol)
+        class DummyProtocol(ProcessProtocol): pass
```

Comment:
One of the things in our coding standard is "no simple suites", so rather than `: pass`, could you make this `<newline><indent>"simple docstring"`?